src/liballoc/string.rs

```diff
@@ -1388,7 +1388,7 @@ impl String {
         self.vec.len()
     }
 
-    /// Returns `true` if this `String` has a length of zero.
+    /// Returns whether this `String` has a length of zero.
```

Comment:
Example of a change affecting public-facing documentation. That feels to me like a @rust-lang/docs decision.

Reply:
Fair point. The quality of public-facing documentation in rustc currently isn't great (at least compared to the stdlib and some libraries), but it would be nice if it reaches that level some day, and if the style is consistent with the stdlib.